ui/app/templates/components/allocation-row.hbs

```diff
@@ -11,8 +11,12 @@
   <td><a href="{{href-to "nodes.node" allocation.node}}">{{allocation.node.shortId}}</a></td>
 {{else if (eq context "node")}}
   <td>
-    <a href="{{href-to "jobs.job" allocation.job}}">{{allocation.job.name}}</a>
-    <span class="is-faded">/ {{allocation.taskGroup.name}}</span>
+    {{#if (or allocation.job.isPending allocation.job.isReloading)}}
+      ...
```

Comment:
I know we don't want too much animation inline, but I didn't realize this was the loading state from the screen caps - would something like bulma's `is-loading` button spinner work?

Reply:
I considered a spinner, but decided it would be distracting. Ideally, people focus on the content that has already loaded until the rest loads. A spinner would pull attention away from that.

I was also going to make a fancier skeleton screen, ala facebook and slack, but decided that would be more effort than it was worth right now. I could still do that though.

I defer to @captainill.